### DISCUSSION TITLE
bigfix/11.0/compliance/Compliance/CarbonBlack/User_Guide/BFCB_landing

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#29] [Q&A]
# bigfix/11.0/compliance/Compliance/CarbonBlack/User_Guide/BFCB_landing

The HCL BigFix and Carbon Black integration allows administrators to deploy a full endpoint security solution to detect, contain, investigate, and remediate security threats and attacks on endpoints across the enterprise. The following sections provide useful information and links to the resources available for the solution.

https://help.hcltechsw.com/bigfix/11.0/compliance/Compliance/CarbonBlack/User_Guide/BFCB_landing.html

**Comment by sweekumari** [comment]
check1
